modules/ding_react/ding_react.module

```diff
@@ -1,5 +1,10 @@
 <?php
 
+define('DING_REACT_FOLLOW_SEARCHES_PROD_URL', 'https://prod.followsearches.dandigbib.org');
+define('DING_REACT_FOLLOW_SEARCHES_STAGE_URL', 'https://stage.followsearches.dandigbib.org');
+define('DING_REACT_MATERIAL_LIST_PROD_URL', 'https://prod.materiallist.dandigbib.org');
+define('DING_REACT_MATERIAL_LIST_TEST_URL', 'https://test.materiallist.dandigbib.org');
```

Comment:
Prod URLs is not used?

Reply:
@xendk Not at the moment. I considered including `ding_react.module` in the update hook and then using the constants. Would you prefer that?

Another approach would be to update the admin UI with radio buttons and an "other" option but that is not something we support in general at the moment.

Even if we do not do anything else I think it is nice to keep them around for reference.

Reply:
@kasperg 
In general I'm in favor of not giving site admins text fields for service URLs but rather radios for environments. The set of valid inputs is usually extremely low, so it's good UX to limit it down, eliminating the risk of typos.

That said, it is important to have a back door for development/debugging purposes, but I'm not adverse to `drush vset ding_react_material_list_url ...`.

Reply:
I agree and appreciate the comment but for this project one does not simply run Drush. On some hosting environments developers do not have direct access to Drush.

Consequently I'll let the current implementation stand.

Reply:
Well, that was the point, you'll only need drush locally. On prod/staging/etc. it should use the prod or test version of the service anyway.